packages/react/package.json

```diff
@@ -46,12 +47,17 @@
     "sass": "^1.75.0",
     "stylelint": "15.1.0",
     "tslib": "^2.6.2",
-    "typescript": "^5.4.5"
+    "typescript": "5.1.6"
```

Comment:
Is this intentional?

Reply:
caret `^` symbol will update  to all future minor/patch versions, without incrementing the major version. ([npm docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies))
since current eslint version supports only upto typescript 5.2, need to avoid this.